src/cmd/cli/command/commands.go

```diff
@@ -181,8 +190,8 @@ func SetupCommands(ctx context.Context, version string) {
 	RootCmd.PersistentFlags().StringArrayP("file", "f", []string{}, `compose file path`)
 	_ = RootCmd.MarkPersistentFlagFilename("file", "yml", "yaml")
 
-	// Setup tracking client
-	track.Fabric = cli.NewGrpcClient(ctx, cluster)
+	// Client a temporary gRPC client for tracking events before login
+	_ = cli.NewGrpcClient(ctx, cluster)
```

Comment:
nit: assignment not needed.
```suggestion
	cli.NewGrpcClient(ctx, cluster)
```

Reply:
Yes but that code without assignment looks wrong